### PR TITLE
Upgrade gRPC and protoc version for aarch support

### PR DESCRIPTION
**Pull request by ZacBlanco**
## Description

Upgrade gRPC and protobuf versions

## Motivation and Context

The older versions didn't support Apple Silicon macs because the binaries for osx-aarch64 were not in the maven repository. Newer versions build these artifacts, which allows builds on these newer Macs to succeed.

## Impact

Unsure of the impact on usage of `presto-grpc-api` and `presto-grpc-testing-udf-server`. Allows full builds to succeed via `./mvnw clean install` without a module argument (`-pl`) on Apple Silicon machines which is a better experience for first-time contributors.

## Test Plan

Existing tests

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-bigquery/pom.xml [hunk 1]
```diff
@@ -73,6 +73,24 @@
                 <artifactId>commons-codec</artifactId>
                 <version>1.13</version>
             </dependency>
+
+            <dependency>
```
Comment (jaystarshot) [inline presto-bigquery/pom.xml:77]: Why is this new dependency added?

File: presto-bigquery/pom.xml [hunk 2]
```diff
@@ -73,6 +73,24 @@
...
+            </dependency>
+
+            <dependency>
+                <groupId>io.grpc</groupId>
+                <artifactId>grpc-protobuf-lite</artifactId>
+                <version>1.64.0</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.perfmark</groupId>
```
Comment (yingsu00) [inline presto-bigquery/pom.xml:90]: io.perfmark doesn't seem to be related, was there a reason why it is needed?

**Review by yingsu00 — COMMENTED** [review]
This is great! Thanks @ZacBlanco for fixing the build errors! Like @jaystarshot I also have a question why io.perfmark was needed. The other dependencies seem related but I'm not sure required.

**Comment by ZacBlanco** [comment]
@yingsu00 @jaystarshot perfmark is a transitive dependency of bigquerystorage from the google cloud libs and gRPC. If you try to build without perfmark pinned to 0.26 you'll get the following error:

```
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for io.perfmark:perfmark-api:0.19.0 paths to dependency are:
+-com.facebook.presto:presto-bigquery:0.288-SNAPSHOT
  +-com.google.cloud:google-cloud-bigquerystorage:1.8.0
    +-io.perfmark:perfmark-api:0.19.0
and
+-com.facebook.presto:presto-bigquery:0.288-SNAPSHOT
  +-com.google.api:gax-grpc:1.60.1
    +-io.grpc:grpc-netty-shaded:1.64.0 (managed) <-- io.grpc:grpc-netty-shaded:1.32.2
      +-io.perfmark:perfmark-api:0.26.0
```

The bigquerystorage dependency from from the import of the google cloud libraries bom

```
            <dependency>
                <groupId>com.google.cloud</groupId>
                <artifactId>libraries-bom</artifactId>
                <version>16.3.0</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```

File: presto-bigquery/pom.xml [hunk 3]
```diff
@@ -73,6 +73,24 @@
...
+                <version>1.64.0</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.grpc</groupId>
+                <artifactId>grpc-protobuf-lite</artifactId>
+                <version>1.64.0</version>
+            </dependency>
+
+            <dependency>
```
Comment (yingsu00) [inline presto-bigquery/pom.xml:89]: Thanks for explaining @ZacBlanco . Would it be possible to add a simple comment here? Something like "required by xxx"?

**Review by yingsu00 — APPROVED** [review]
Looks good. I'll merge once the comment is added.

File: presto-bigquery/pom.xml (hunk 3 above)
Comment (ZacBlanco) [reply presto-bigquery/pom.xml:91]: added

**Review by yingsu00 — APPROVED** [review]
